olive/workflows/run/config.py

```diff
@@ -402,6 +402,9 @@ def _resolve_system(v, values, system_alias):
 
 
 def _resolve_data_config(v, values, data_config_alias):
+    if isinstance(data_config_alias, dict):
```

Comment:
Can you also check for DataConfig object to cover all possible cases?

Reply:
Please also make the error more informative. Something like `"Inline data configs are not supported! Use the name of a data config defined under 'data_configs'"`